requirements-dev.txt

```diff
@@ -0,0 +1,7 @@
+-r requirements.txt
+flake8==3.0.4
+mock==2.0.0
+pytest==3.0.3
+# Weather
```

Comment:
>Just a rebase now

sorry i'm still beginner in git. do you mean rebase is actually squash the commit and make this branch look like this ?

https://github.com/rachmadaniHaryono/Melissa-Core/tree/add-stt-tts-module-tests-single